File: mockserver-client-php/src/Llm/Provider.php
<?php

declare(strict_types=1);

namespace MockServer\Llm;

/**
 * LLM provider names. Serialized on the wire as the upper-case enum name
 * (mirrors org.mockserver.model.Provider).
 */
final class Provider
{
    public const ANTHROPIC = 'ANTHROPIC';
    public const OPENAI = 'OPENAI';
    public const OPENAI_RESPONSES = 'OPENAI_RESPONSES';
    public const GEMINI = 'GEMINI';
    public const BEDROCK = 'BEDROCK';
    public const AZURE_OPENAI = 'AZURE_OPENAI';
    public const OLLAMA = 'OLLAMA';

    private function __construct()
    {
    }
}

File: mockserver/mockserver-core/src/main/java/org/mockserver/model/Provider.java
package org.mockserver.model;

public enum Provider {
    ANTHROPIC,
    OPENAI,
    OPENAI_RESPONSES,
    GEMINI,
    BEDROCK,
    AZURE_OPENAI,
    OLLAMA,
    COHERE,
    VOYAGE,
    // OpenAI-chat-compatible providers: identical Chat Completions wire format on a
    // different host, so their codecs/clients delegate to the OpenAI implementations
    // and are distinguished only by host (see LlmProviderSniffer / ProviderDetector).
    MISTRAL,
    XAI,
    DEEPSEEK,
    GROQ,
    OPENROUTER
}
